test/module/shared_model/validators/batch_validator_test.cpp

```diff
@@ -97,3 +97,22 @@ TEST_F(BatchValidatorFixture,
               ::testing::HasSubstr("Hashes of provided transactions and ones "
                                    "in batch_meta are different"));
 }
+
+/**
+ * @given a batch validator with allowed partial ordered batches
+ * @when an ordered batch with some transactions missing and the rest of
```

Comment:
Description is copy-pasted from the test case above, please fix.

Reply:
Fixed